src/main/java/seedu/mark/logic/Logic.java

```diff
@@ -51,4 +53,19 @@
      * Set the user prefs' GUI settings.
      */
     void setGuiSettings(GuiSettings guiSettings);
+
+    /**
+     * Wrapper for current url.
+     * null if not present.
+     *
+     * @see seedu.mark.model.Model#currentUrlProperty()
+     */
+    SimpleObjectProperty<Url> currentUrlProperty();
```

Comment:
```suggestion
    SimpleObjectProperty<Url> getCurrentUrlProperty();
```

Since here there's `getMark` to get the mark

Reply:
Ok